packages/gatsby-theme-docs/gatsby-ssr.js

```diff
@@ -3,5 +3,23 @@
  *
  * See: https://www.gatsbyjs.org/docs/ssr-apis/
  */
+import React from 'react';
 
-// You can delete this file if you're not using it
+// eslint-disable-next-line import/prefer-default-export
+export const onRenderBody = ({ setPostBodyComponents }) => {
```

Comment:
@nkuehn FYI: this was the problem:

```js
// works
export const

// does not work
export function
```

https://github.com/gatsbyjs/gatsby/issues/4718